customer.go

```diff
@@ -205,7 +205,12 @@ func (c *Client) CreateCustomer(customer *Customer, requestID string) (*Customer
 	}
 	u.Path = "/v3/company/" + c.RealmID + "/customer"
 	if requestID != "" {
-		u.Path += "?requestid=" + requestID
+		query, err := url.ParseQuery(u.RawQuery)
```

Comment:
Instead of re-parsing the raw query you can use `u.Query()` to get the already parsed values.

```golang
query := u.Query()
query.Add("requestid", requestID)
u.RawQuery = query.Encode()
```